dockerfiles/circleci/Dockerfile

```diff
@@ -16,8 +16,8 @@ RUN export CLOUD_SDK_REPO="cloud-sdk-$(lsb_release -c -s)" && \
   curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add - && \
   sudo apt-get update -y && sudo apt-get install google-cloud-sdk -y
 
-RUN sudo wget https://dl.google.com/go/go1.11.5.linux-amd64.tar.gz && \
-  sudo tar xf go1.11.5.linux-amd64.tar.gz -C /usr/local
+RUN sudo wget https://dl.google.com/go/go1.13.5.linux-amd64.tar.gz && \
```

Comment:
Is 1.13.5 the exact version we for sure want? Just double checking because this looks like it could have been just a change of the minor version.